### PR TITLE
chore(release): v2.7.0

### PR DESCRIPTION
**Pull request by blushi**
## Description

**DO NOT MERGE FROM GITHUB**
see https://www.notion.so/regennetwork/Release-process-for-regen-web-3dce8a42813b466fa46d44e8723a3b62

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by clevinson — APPROVED** [review]

**Comment by erikalogie** [comment]
When I went to delete a post from the post page I got an "error: Ok" message. I think this should redirect to the data stream with this message: https://www.figma.com/design/Bksz1JeDYxQVIXdI46EgPT/Project-Posts?node-id=4608-211484&t=p6ULrokKt5lVWWnY-1


https://github.com/user-attachments/assets/25227af5-6040-42ce-ae86-2200cd03a3e0

**Comment by blushi** [comment]
> When I went to delete a post from the post page I got an "error: Ok" message. I think this should redirect to the data stream with this message: https://www.figma.com/design/Bksz1JeDYxQVIXdI46EgPT/Project-Posts?node-id=4608-211484&t=p6ULrokKt5lVWWnY-1
> 
>  Screen.Recording.2024-09-11.at.9.39.54.AM.mov

I'll have a look, I've already tagged a release so I'll need to make a hot fix

File: web-www/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@mdx-js/react": "^2.3.0",
     "@mui/material": "^5.10.10",
     "@mui/styles": "^5.10.10",
-    "@netlify/plugin-nextjs": "4.41.3",
+    "@netlify/plugin-nextjs": "5.7.0",
     "@next/mdx": "^13.4.19",
```
Comment (paul121) [inline web-www/package.json:26]: I see this was bumped to a new major version. There are a couple breaking changes, not sure if these are related: https://docs.netlify.com/frameworks/next-js/overview/#breaking-changes

cc @clevinson

Comment (paul121) [reply web-www/package.json:26]: This was changed here in #2408  https://github.com/regen-network/regen-web/pull/2408/commits/bc73188de37115ca680eb6656b96d458174a2eab It looks like all previews have been failing after this was merged ~12 hours ago.

I think going back to v4 of `netlify/plugin-nextjs` might fix the problem?

**Pull request merged**